pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -105,6 +105,21 @@ func GetPodInterfaceFromCache(uid types.UID, ifaceName string) (*PodCacheInterfa
 	return result, nil
 }
 
+func PutPodInterfaceInCache(uid types.UID, ifaceName string, iface PodCacheInterface) error {
```

Comment:
We can have two different methods, one for virt-handler cache files and one for virt-launcher cache files. But since currently the two methods are basically a copy paste (besides the path part), I think the duplicated code (writing data to cached file on a specified path) should be extracted to a common method that both of the methods will use.

Also, I find it confusing to have both `PutPodInterfaceInCache` and `setPodInterfaceInCache` methods.
Instead of `PutPodInterfaceInCache` I would call the method `WriteToVirtHandlerCachedFile`. The method will write to `virtutil.VMIInterfacepath` + the suffix passed as a parameter to the method.

So to conclude I suggest to have `WriteToVirtHandlerCachedFile`, `WriteToVirtLauncherCachedFile` and `writeToCachedFile`.